Sources/RInAppMessaging/Views/Presenters/FullViewPresenter.swift

```diff
@@ -98,4 +122,15 @@ internal class FullViewPresenter: BaseViewPresenter, FullViewPresenterType {
         logImpression(type: .optOut)
         optOutCampaign()
     }
+
+    private func pushPrimerAction() {
+        notificationCenter.requestAuthorization(options: pushPrimerOptions) { [weak self] (granted, error) in
```

Comment:
Is the push primer callback not included?

Reply:
It looks like on iOS there will be a different approach. The SDK will be always handling push registration process. The only control from SDK user perspective is by changing the authorizationOptions

Reply:
You mean the option 3 is no longer possible?

Reply:
It is possible. That's just PDM's decision to go with Option 4

Reply:
Got it.
Thank you for the explanation.